demofile-parsing/parse_demo.js

```diff
@@ -0,0 +1,25 @@
+const fs = require("fs");
+const demofile = require("demofile");
+const path = require("path");
+
+fs.readFile(path.resolve("/Users/tylerlam/Downloads/test.dem"), (err, buffer) => {
```

Comment:
other than this abomination, clean, concise, a good start <3

Reply:
true I'll update that boii next PR